installers/rpm-code/codes/geant4.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+: notes <<'EOF'
+
+If you see the following errors try setting LIBGL_ALWAYS_INDIRECT=1
+libGL error: No matching fbConfigs or visuals found
+libGL error: failed to load driver: swrast
+https://unix.stackexchange.com/questions/589236/libgl-error-no-matching-fbconfigs-or-visuals-found-glxgears-error-docker-cu
+https://unix.stackexchange.com/questions/1437/what-does-libgl-always-indirect-1-actually-do
+
+To help debug libgl errors:
+LIBGL_DEBUG=verbose
+
+On a Mac if the geant4 GUI appears but the space where the visualization should be is blank you may
+need to enable opengl:
+https://geant4-forum.web.cern.ch/t/qt-viewer-scene-tree-empty/1207/5 (which links to)
+https://services.dartmouth.edu/TDClient/1806/Portal/KB/ArticleDet?ID=89669
+EOF
+
+geant4_install_data_download_script() {
+    install -m 555 /dev/stdin "${codes_dir[bin]}"/rs_geant4_download_data <<'EOF'
+#!/bin/bash
+set -eou pipefail
```

Comment:
Since this is a command that is installed somewhere, a header comment with usage and description is useful. One way to do this is:

```sh
#/bin/bash
#
# Install data sets for Geant4 from https://cern.ch/geant4-data/datasets
#
_USAGE="usage: $0 dataset-install-directory
You must supply a directory where you want the data sets to be installed"

Reply:
1564945